tests/cephfs/cephfs_pool_pg_size_change.py

```diff
@@ -0,0 +1,213 @@
+import datetime
+import logging
+import secrets
+import string
+import time
+import traceback
+
+from ceph.ceph import CommandFailed
+from ceph.parallel import parallel
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+logger = logging.getLogger(__name__)
+log = logger
+
+
+def check_clean_pgs(client, timeout=300):
+    """
+    Verify all pgs are active+clean
+    :param client: client node
+    :param timeout: timeout for checking pg state
+    """
+    timeout = datetime.timedelta(seconds=timeout)
+    starttime = datetime.datetime.now()
+    out, rc = client[0].exec_command(
+        sudo=True, cmd="ceph pg stat | awk '{{print $1 }}'"
+    )
+    total_pgs = out.read().decode()
+    while datetime.datetime.now() - starttime <= timeout:
+        out, rc = client[0].exec_command(
+            sudo=True, cmd="ceph pg stat | awk '{{print $3 }}'"
+        )
+        active_clean_pgs = out.read().decode()
+        out, rc = client[0].exec_command(
+            sudo=True, cmd="ceph pg stat | awk '{{print $4 }}'"
+        )
+        output = out.read().decode()
```

Comment:
It is expensive to issue multiple SSH commands to retrieve the value. Switching to `--format json` output would help get all the information in a single SSH command. This reduces the time and no of failure points.